source/CommonJobs/CommonJobs.MVC.UI/Areas/EvalForm/Controllers/EvalFormController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using CommonJobs.Application.MyMenu;
+using CommonJobs.Domain;
+using CommonJobs.Domain.MyMenu;
+using CommonJobs.Infrastructure.Mvc;
+using CommonJobs.Mvc.UI.Infrastructure;
+using NLog;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using CommonJobs.Utilities;
+
+namespace CommonJobs.Mvc.UI.Areas.EvalForm
+{
+    [CommonJobsAuthorize]
+    //TODO: Add documentation
+    [Documentation("docs/manual-de-usuario/evaluaciones")]
+    public class EvalFormController : CommonJobsController
+    {
+        private static Logger log = LogManager.GetCurrentClassLogger();
+
+        public ActionResult Index()
+        {
+            //TODO: ...
+            return null;
```

Comment:
Also, it is not breaking anything.
El 10/06/2015 23:01, "Alpha" <notifications@github.com> escribió:

> In
> source/CommonJobs/CommonJobs.MVC.UI/Areas/EvalForm/Controllers/EvalFormController.cs
> <https://github.com/CommonJobs/CommonJobs/pull/375#discussion_r32184602>:
>
> > +using System.Web.Mvc;
> > +using CommonJobs.Utilities;
> > +
> > +namespace CommonJobs.Mvc.UI.Areas.EvalForm
> > +{
> > +    [CommonJobsAuthorize]
> > +    //TODO: Add documentation
> > +    [Documentation("docs/manual-de-usuario/evaluaciones")]
> > +    public class EvalFormController : CommonJobsController
> > +    {
> > +        private static Logger log = LogManager.GetCurrentClassLogger();
> > +
> > +        public ActionResult Index()
> > +        {
> > +            //TODO: ...
> > +            return null;
>
> Does this controller work, or is this still work in progress?
>
> As a personal opinion, I feel against merging to the main branch of the
> repository code that is not yet finished -- that's why I'm asking. Again,
> it is just a personal opinion so let me know how do you want to approach
> this, since any non-finished part adds risk to the product of being
> released / updated later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CommonJobs/CommonJobs/pull/375/files#r32184602>.
>